src/cljs/org/broadinstitute/firecloud_ui/paths.cljs

```diff
@@ -45,5 +45,8 @@
 (defn submission-details [workspace-id submission-id]
   (str (submissions-list workspace-id) "/" submission-id))
 
+(defn get-abort-submission-path [workspace-id submission-id]
+  (submission-details workspace-id submission-id))
+
```

Comment:
this function calls the other directly.
however another name shows it can serve dual purposes. (DELETE and GET as you mention).
However, "submission-details" when called,might possibly be confusing in a DELETE contact, so that was the motivation to make something, with a different (maybe less confusing) name to call the other function ( a few lines ahead)